threatmodel.md

```diff
@@ -0,0 +1,16 @@
+## Threat Model Assessment
```

Comment:
Good job in identifying the threats, but the main question would be what measures your team will take in order to mitigate those attacks/hacks/threats.